src/common/include/com/amazonaws/kinesis/video/common/CommonDefs.h

```diff
@@ -201,6 +201,13 @@ typedef FLOAT* PFLOAT;
 #define TRUE 1
 #endif
 
+// Syslog
+#if defined _WIN32 || defined _WIN64
+#error Windows does not support syslog

```

Comment:
#error would lead to failed execution on Windows. I would suggest having a compile time definition to enable and disable syslogger based on platform. 
This would help in getting rid of the CI failure.

Reply:
Perhaps I am not reading this right but this actually will break Windows builds in the first place. We need to actually make it somehow a compile-time syslog-based usage only failure. Need to think about the best solution but at the end, we could use a dynamic fallback or something. For example, the Windows implementation could simply do ETW tracing or at worst end, simple standard output - default to printf?

Reply:
Right #error is causing the build to fail, all other CI is passing, @thinkski if you can fix this we can have this merged in. 

Error message:  
```
C:\Users\travis\build\awslabs\amazon-kinesis-video-streams-pic\src\common\include\com/amazonaws/kinesis/video/common/CommonDefs.h(206): fatal error C1189: #error:  Windows does not support syslog
The command ".github/build_windows.bat" exited with 2.
```

Reply:
Fixed. Windows build is still failing CI, but appears to be an unrelated issue:
```
[ RUN      ] PermutatedStreamInfo/IntermittentProducerFunctionalityTest.CreateSyncStreamWithLargeBufferAwaitForLastAckStopSyncFreeSuccess/6
c:\users\travis\build\awslabs\amazon-kinesis-video-streams-pic\src\client\tst\ClientTestFixture.h(664): error: Value of: STATUS_SUCCESS == mThreadReturnStatus
  Actual: false
Expected: true
c:\users\travis\build\awslabs\amazon-kinesis-video-streams-pic\src\client\tst\ClientTestFixture.h(665): error: Value of: mStreamingSession.mConsumerList.empty()
  Actual: false
Expected: true
[  FAILED  ] 
```